components/LeagueEntries/LeagueEntries.test.tsx

```diff
@@ -0,0 +1,64 @@
+import { LeagueEntries } from './LeagueEntries';
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+
+describe('LeagueEntries', () => {
+  it('renders without a pick made', () => {
+    render(<LeagueEntries entryNumber={1} />);
+
+    const leagueEntryContainerCard = screen.getByTestId(
+      'league-entry-container-card',
+    );
+    const leagueEntryNumber = screen.getByTestId('league-entry-number');
+    const entryStatus = screen.getByTestId('entry-status');
+    const leagueEntryPickButton = screen.getByTestId('league-entry-pick-button');
+
+    expect(entryStatus).toBeInTheDocument();
+    expect(entryStatus).toHaveTextContent('alive');
+    expect(leagueEntryContainerCard).toBeInTheDocument();
```

Comment:
No need to check for this as there's no condition to remove this element. It's always going to be there.

Reply:
Removed in latest commit